memote/suite/tests/test_thermodynamics.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 Novo Nordisk Foundation Center for Biosustainability,
+# Technical University of Denmark.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Perform tests using eQuilibrator API on an instance of ``cobra.Model``."""
+
+from __future__ import absolute_import, division
+
+from sys import version_info
+
+import pytest
+
+if version_info[:2] < (3, 5):
+    pytest.skip("Thermodynamic tests require at least Python version 3.5.",
+                allow_module_level=True)
+
+import memote.support.thermodynamics as thermo # noqa
+import memote.support.basic as basic # noqa
+from memote.utils import (annotate, get_ids, truncate) # noqa
+
+
+@annotate(title="Thermodynamic Reversibility of Purely Metabolic Reactions",
+          format_type="percent")
+def test_find_incorrect_thermodynamic_reversibility(read_only_model):
+    u"""
+    Expect reversibility of metabolic reactions to agree with thermodynamics.
+
+    If a reaction is neither a transport reaction, a biomass reaction nor a
+    boundary reaction, it is counted as a purely metabolic reaction.
+    This test checks if the reversibility attribute of each reaction
+    in a list of cobrapy reactions agrees with a thermodynamics-based
+    calculation of reversibility. To determine reversibility we calculate
+    the reversibility index lngamma of each reaction
+    using the eQuilibrator API. The default cutoff for lngamma of 3
+    "corresponds to allowing concentrations to span three orders of magnitude
+    around 100 μM (~3 μM—3mM)" at "pH = 7, I = 0.1M and T = 298K". This means
```

Comment:
Is it immediately obvious to everyone what _pH_, _I_, and _T_ stand for?

Reply:
I think every student of natural sciences will have had the basics of physics and chemistry to understand pH and T (especially when the value is above 200 and is in kelvin), but I do concede that `I` is still cryptic to me. I'll add an explanation.